### DISCUSSION TITLE
Converting from .ipynb to .md

### DISCUSSION THREAD

**Discussion by oye-bobs** [cu-esiil-edu/2024-data-short-course#60] [Q&A] [answered]
Hi @nquarder , I tried converting the jupyter notebook .ipynb file to a markdown .md file using this code
![Screenshot 2024-04-22 122128](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/129c96ed-24e9-4617-99a8-515284ffd1e4)

 but it kept giving an error. How do I fix it?

**Comment by oye-bobs** [comment]
@nquarder I keep getting this error whenever I run the code to convert from ipynb to html
![Screenshot 2024-04-24 104459](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/b905a5dd-e311-49ad-9cb2-7206ebb47bce)

**Comment by nquarder** [comment]
@oye-bobs have you tried repeating this but without the quotations marks around the .ipynb file name? 

Give that a try and see if it helps. -NQ

**Reply by oye-bobs** [reply, marked as answer]
Tried that, but It didn't work. However, I was able to fix it by using "!"  to execute it as a shell command
![4](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/5c1ec3d4-0059-4203-b65a-8fb0d14f15cc)
